data-prepper-plugins/opensearch/src/main/java/org/opensearch/dataprepper/plugins/sink/opensearch/BulkRetryStrategy.java

```diff
@@ -209,7 +209,7 @@ public void execute(final AccumulatingBulkRequest bulkRequest) throws Interrupte
                 request = operationResponse.getBulkRequest();
                 exceptionMessage = operationResponse.getExceptionMessage();
                 if (delayMillis < 0) {
-                    RuntimeException e = new RuntimeException(String.format("Index : %s, Number of retries reached the limit of max retries (configured value %d. Last exception message: %s)", maxRetries, exceptionMessage));
```

Comment:
Why get rid of the index name here?

Reply:
@graytaylor0 I did not get rid of it. It is not available. I have mistakenly put it there in the previous revision.

Reply:
Index is available at each operation level. Not at bulk operation level.